src/virtual_channel/virtual_channel.c

```diff
@@ -105,7 +108,11 @@ size_t get_virtual_channel_count(void)
 
 void reset_virtual_channels(void)
 {
+        xSemaphoreTake(vchan_mutex, portMAX_DELAY);
         g_virtualChannelCount = 0;
+        xSemaphoreGive(vchan_mutex);
+	reset_logger_config();
```

Comment:
I don't think we want to reset the logger config.  The virtual channels are reset when the script re-loads, but when the script reloads we don't want to wipe the rest of the logger config. 

I think triggering the configChanged() is good, however.

Reply:
I'll make that change and retest tomorrow.